task/control-plane-agent/src/mgs_sidecar.rs

```diff
@@ -312,6 +316,14 @@ impl MgsHandler {
                 UnlockChallenge::Trivial { timestamp: ts1 },
                 UnlockResponse::Trivial { timestamp: ts2 },
             ) if ts1 == ts2 => Ok(()),
+            (
```

Comment:
Tiny nit - The comment just outside the diff above this is out of date ("right now there's only one option..."). It's maybe also wrong - since we use a `_ => Err(_)` the match doesn't really check for new types?

Reply:
Fixed!